php_phongo.c

```diff
@@ -2160,6 +2160,8 @@ static mongoc_ssl_opt_t* php_phongo_make_ssl_opt(mongoc_uri_t* uri, zval* zoptio
 	} else if (php_array_existsc(zoptions, "capath")) {
 		PHONGO_SSL_OPTION_SWAP_STRING(ssl_opt->ca_dir, "capath");
 		any_ssl_option_set = true;
+
+		php_error_docref(NULL TSRMLS_CC, E_DEPRECATED, "The \"capath\" driver option is deprecated. Please use the \"ca_dir\" driver option instead.");
```

Comment:
I think we should use a more explicit message for options that were [merged from the "context" option](https://github.com/mongodb/mongo-php-driver/blob/1.6.0/src/MongoDB/Manager.c#L47). Consider:

> The "capath" context driver option is deprecated...